src/Microsoft.Health.Dicom.SqlServer/Features/CustomTag/CustomTagLimit.cs

```diff
@@ -0,0 +1,41 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+using System.Collections.Generic;
+using Dicom;
+
+namespace Microsoft.Health.Dicom.SqlServer.Features.CustomTag
+{
+    /// <summary>
+    /// Limits on CustomTag feature.
+    /// </summary>
+    public static class CustomTagLimit
```

Comment:
Should this exist outside of the `SqlCustomTagStore`? I don't think any other class/component is going to use this mapping

Reply:
shoudl be internal